data/binding/gen.go

```diff
@@ -1011,6 +1114,13 @@ import (
 
 	"fyne.io/fyne/v2"
 )
+
+func internalFloatToInt(val float64) (int, error) {
+	return int(val), nil
+}
+func internalIntToFloat(val int) (float64, error) {
```

Comment:
This needs a newline

Reply:
Thanks for updating it, but I don't think you re-generated as the two are still compressed in the output...